roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Define RH Version
+  set_fact:
+      rh_ver: "{{ ansible_distribution_version|int|abs }}"
```

Comment:
Is converting to `int` and so on required?
`ansible_distribution_version` shows `7Server` for you?